java/src/main/c++/jni_spark_cluster.cc

```diff
@@ -0,0 +1,63 @@
+#include "jni_spark_vw_generated.h"
+#include "util.h"
+#include "spanning_tree.h"
+
+JNIEXPORT jlong JNICALL Java_org_vowpalwabbit_spark_ClusterSpanningTree_create
+  (JNIEnv *env, jclass, jint port)
+{
+  try

```

Comment:
Could you format this based on the .clang-format rules?

Reply:
done